src/SFA.DAS.LevyTransferMatching.Domain/Extensions/PledgeStatusExtenstions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using SFA.DAS.LevyTransferMatching.Domain.Types;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.LevyTransferMatching.Domain.Extensions
+{
+    public static class PledgeStatusExtenstions
+    {
+        public static string GetLabelForPledgeStatus(this PledgeStatus status)
+        {
+            switch (status)
+            {
+                case PledgeStatus.Active: return "ACTIVE";
+                case PledgeStatus.Closed: return "CLOSED";
+                default:
+                    return string.Empty;
+            }
+        }
+
+        public static string GetCssClassForPledgeStatus(this PledgeStatus status)
+        {
+            switch (status)
+            {
+                case PledgeStatus.Active: return "govuk-tag govuk-tag--blue";
```

Comment:
The spec (pic on the story) shows dark blue, but the web page shows light blue background?

Reply:
I have updated this to Dark Blue "govuk-tag govuk-tag--dark-blue";